packages/flutter_tools/templates/app/lib/main.dart.tmpl

```diff
@@ -35,7 +35,7 @@ class MyApp extends StatelessWidget {
 }
 
 class MyHomePage extends StatefulWidget {
-  MyHomePage({Key key, this.title}) : super(key: key);
+  MyHomePage({Key? key, this.title = 'Home Page'}) : super(key: key);
```

Comment:
I would have marked `title` as required. (Partly to also introduce that keyword to developers in case they don't know about it).

Reply:
Yes, i was contemplating that also. @munificent which of the two concepts do you think is most important to show?

Reply:
Please go with required, it's something new/different compared with Python et al.

Reply:
Done